cypress/integration/Crosscutting/HttpStatusCodeTest.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Test for checking HTTP Status Code
+
+  Scenario Outline: Check received status code from get request
+        Given user is navigating to "<path>"
```

Comment:
no need to navigate at all, you can delete this step , sending request in the second step is more than enough

Reply:
done